cmd/kyma/alpha/enable/module/opts.go

```diff
@@ -1,19 +1,26 @@
 package module
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/kyma-project/cli/internal/cli"
 	"github.com/pkg/errors"
 )
 
+const (
+	CustomResourcePolicyCreateAndDelete = "CreateAndDelete"
```

Comment:
Since these consts are not used outside of it (for now), I guess they can be package-private.